web/html/src/branding/css/base/setup-wizard.less

```diff
@@ -247,6 +247,10 @@ ul.product-list {
         margin: 0 auto;
       }
     }
+
+    .calc-width {
```

Comment:
While I know this is how the class was named before, perhaps we could find a more descriptive name now that we're using it? Since every column where this is used is for the description field as far as I can see, maybe a name on those lines, e.g. col-description or something similar?

Reply:
Makes sense. I will change it  to something more descriptive